configure.ac

```diff
@@ -3,6 +3,7 @@
 
 AC_PREREQ([2.69])
 AC_INIT([powertop], [2.14], [powertop@lists.01.org], [], [https://01.org/powertop])
+AC_CANONICAL_TARGET
```

Comment:
This is almost never what you want to use. See https://flameeyes.blog/2008/10/12/you-dont-want-to-canonicalise-that-and-im-not-referring-to-ubuntu/

Reply:
I suspect you added this because `AX_PTHREAD` appeared to need it. This was a bug and was fixed in https://github.com/autoconf-archive/autoconf-archive/pull/233. See the discussion in https://github.com/autoconf-archive/autoconf-archive/commit/2567e0ce0f3a11b535c6b527386197fb49ff172b

Unfortunately, while the revert landed in master months ago the autoconf-archive maintainer has not made another release, despite multiple pings, thereby confirming our worries that this was going to trigger problems downstream.

Reply:
Yep, thanks for info.